src/editor_view.hpp

```diff
@@ -65,10 +65,14 @@ public slots:
 
     void setEdgeColor(const QColor & edgeColor);
 
+    void setGridColor(const QColor & edgeColor);
+
     void setEdgeWidth(double edgeWidth);
 
     void setGridSize(int size);
 
+    void setGridVisible(int state);
```

Comment:
Having `int state` is very confusing here. The parameter should be `bool visible` etc and you could then compare the state with Qt::Checked outside of this function.